tox.ini

```diff
@@ -196,6 +196,8 @@ markers =
     timeout: tests that need a timeout time
     localtest: local tests
     scala: scala tests
+    modin_sp_short_regress: modin short_regress tests run in sproc
```

Comment:
add a comment here to make it clear that those marker are used at server side to decide which tests to run, any change to the name will require a server change also, please be careful with any changes here

Reply:
Is this obvious? If marker changed, it definitely will affect server side.

Reply:
I don't think it is obvious, this is client side change, it is just used to run the tests, can be at both client and server. If  any change is made at client side, server side test will not automatically run,  which may not be captured immediately. 
It would be better if we can add a comment here to make it more clear to all developer in Snowpark python

Reply:
I see, we can add more comments to make it clear.